crates/re_renderer/shader/decodings.wgsl

```diff
@@ -0,0 +1,19 @@
+#import <./types.wgsl>
+
+
+/// Loads an RGBA texel from a texture holding an NV12 encoded image at the given screen space coordinates.
```

Comment:
is there a source link or reference doc you can provide for the factors for the conversion?

Reply:
Yes! I added it directly to the code. They are Bt.601 coefficients, which is what we use for our cameras at [luxonis](https://www.luxonis.com/). I also added the conversion using BT.709 coefs. Now.. we may want to expose this, to let the user choose the standard, but I think for the majority just using one will be good enough, there will just be a very slight difference in colors.
Convenient online example :http://www.glennchan.info/articles/technical/rec709rec601/rec709rec601.html

Reply:
ah nice, thank you!
But would than mean that for all the luxonis usecases you'd need to uncomment that code? If so we should fine a nice way to expose this right away.

Reply:
Ah, yes, I actually wanted to commit the Bt.601 decodeing uncommented, but looks like I switched it up when I was testing it before commiting.. I suppose I agree on exposing it right away, take a look ^ at the size_hint discussion :)